runtests.sh

```diff
@@ -139,9 +140,15 @@ function run_test()
     local correct_exit_code="$2"
     local counter="$3"
 
-    local command diffpath
-    command="$(printf "$command_template" $joufile)"
-    diffpath=tmp/tests/diff$(printf "%04d" $counter).txt  # consistent alphabetical order
+    local command
+    if [[ "$joufile" =~ ^examples/aoc ]]; then
+        # AoC files use fopen("sampleinput.txt", "r")
```

Comment:
```suggestion
        # AoC files use fopen("sampleinput.txt", "r").
        # We don't do this for all files, because I like relative paths in error messages.
```